assets/scripts/outputs.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -e
+
+# set filename for storing variables and values
+output_file="output.txt"
+
+# set header and footer strings
+header="###pipeline-summary-start###"
+footer="###pipeline-summary-end###"
+
+# check if file exists, if not create it
+if [ ! -f "$output_file" ]; then
+    touch "$output_file"
+fi
+
+function help() {
+    script_name="$0"
+    echo "Usage: $script_name add <variable name> <value>"
+    echo "Usage: $script_name run <json string>"
+    echo "Usage: $script_name print"
+}
+
+# check if user wants to add a new variable and value
```

Comment:
I don't understand the wording of this comment?
"check if user..."
why "check"?
doesn't it always add a new variable and value?